src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -100,19 +100,19 @@ internal void DestroyContract()
                     Snapshot.Storages.Delete(key);
         }
 
-        internal void CallContract(UInt160 contractHash, string method, Array args)
+        internal void CallContract(UInt160 contractHash, ContractParameterType returnType, string method, Array args)
         {
-            CallContractInternal(contractHash, method, args, CallFlags.All);
+            CallContractInternal(contractHash, returnType, method, args, CallFlags.All);
         }
 
-        internal void CallContractEx(UInt160 contractHash, string method, Array args, CallFlags callFlags)
+        internal void CallContractEx(UInt160 contractHash, ContractParameterType returnType, string method, Array args, CallFlags callFlags)
         {
             if ((callFlags & ~CallFlags.All) != 0)
                 throw new ArgumentOutOfRangeException(nameof(callFlags));
-            CallContractInternal(contractHash, method, args, callFlags);
+            CallContractInternal(contractHash, returnType, method, args, callFlags);
         }
 
-        private void CallContractInternal(UInt160 contractHash, string method, Array args, CallFlags flags)
+        private void CallContractInternal(UInt160 contractHash, ContractParameterType returnType, string method, Array args, CallFlags flags)
```

Comment:
@Tommo-L I won't modify it. Because it keeps the same order with `EmitAppCall()`.

https://github.com/neo-project/neo/blob/061e61d31a0d6ee4c4ab965d643829df865af1e6/src/neo/VM/Helper.cs#L37

Reply:
Okay